packages/experimental/TabList/src/TabListTokens.macos.ts

```diff
@@ -0,0 +1,15 @@
+import type { Theme } from '@fluentui-react-native/framework';
+import type { TokenSettings } from '@fluentui-react-native/use-styling';
+
+import type { TabListTokens } from '.';
+
+export const defaultTabListTokens: TokenSettings<TabListTokens, Theme> = (t: Theme) =>
+  ({
+    color: t.colors.menuItemText,
+    variant: 'bodySemibold',
+    fontWeight: 'bold',
+    fontSize: 14,
```

Comment:
Can you use a token here?

Reply:
For the text styling? I do plan on updating tokens to be in line with web and using tokens here, but I was hoping to keep this PR limited to simply creating the package.

Reply:
For next time, you should separate out prs for defining prop and tokens types, and for setting specific platform style tokens, instead of hard coding it like this and updating it later. It's helpful for when we have to look back at each pr